civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -70,3 +70,16 @@ $row-margin-bottom: 15px;
     }
   }
 }
+
+/*
+* This is a TEMP fix to fit the text of input controls like date picker within input box
+* It is hardcoded to 13px for now.
+*/
+.chr_leave-request-modal__input-font-size {
```

Comment:
do you mean something like `chr_leave-calendar__legend__label` but its name suggests its for calendar?